examples/material_gallery/lib/demo/flexible_space_demo.dart

```diff
@@ -0,0 +1,192 @@
+
+  Widget build(BuildContext context) {
+    return new Theme(
+      data: new ThemeData(
+        brightness: ThemeBrightness.light,
+        primarySwatch: Colors.indigo
+      ),
+      child: new Scaffold(
+        appBarHeight: appBarHeight,
+        scrollableKey: scrollableKey,
+        appBarBehavior: AppBarBehavior.scroll,
+        toolBar: new ToolBar(
+          right: <Widget>[
+            new IconButton(
+              icon: "content/create",
+              tooltip: 'Search'
+            ),
+            new IconButton(
+              icon: "navigation/more_vert",
+              tooltip: 'Show menu'
+            )
+          ],
+          flexibleSpace: (_) {
+            return new FlexibleSpaceBar(
+              title : new Text("Ali Connors"),
+              image: new AssetImage(
+                name: "assets/ali_connors.png",
+                fit: ImageFit.cover,
+                height: appBarHeight
+              )
+            );
+          }
+        ),
+        body: new Block(
+          scrollableKey: scrollableKey,
+          padding: new EdgeDims.only(top: appBarHeight),
+          children: <Widget>[
+            new _ContactCategory(
+              icon: "communication/call",
+              children: <Widget>[
+                new _ContactItem(
+                  icon: "communication/message",
+                  lines: <String>[
+                    "(650) 555-1234",
+                    "Mobile"
+                  ]
+                ),
+                new _ContactItem(
+                  icon: "communication/message",
+                  lines: <String>[
+                    "(323) 555-6789",
+                    "Work"
+                  ]
+                )
+              ]
+            ),
+            new _ContactCategory(
+              icon: "communication/email",
+              children: <Widget>[
+                new _ContactItem(
+                  lines: <String>[
+                    "ali_connors@example.com",
+                    "Personal"
+                  ]
+                ),
+                new _ContactItem(
+                  lines: <String>[
+                    "aliconnors@example.com",
+                    "Work"
+                  ]
+                )
+              ]
+            ),
+            new _ContactCategory(
+              icon: "communication/location_on",
+              children: <Widget>[
+                new _ContactItem(
+                  lines: <String>[
+                    "2000 Main Street",
+                    "San Francisco, CA",
+                    "Home"
+                  ]
+                ),
+                new _ContactItem(
+                  lines: <String>[
+                    "1600 Amphitheater Parkway",
+                    "Mountain View, CA",
+                    "Work"
+                  ]
+                ),
+                new _ContactItem(
+                  lines: <String>[
+                    "126, Severyns Ave",
```

Comment:
Yes. I copied the address of the Moffet Field museum: http://bit.ly/1NSpS69. It includes the extra comma, but I don't think it's right either :-).